pkg/scheduler/factory.go

```diff
@@ -165,7 +165,7 @@ func (c *Configurator) create() (*Scheduler, error) {
 			prof.PluginConfig = append(prof.PluginConfig,
 				frameworkplugins.NewPluginConfig(
 					noderesources.FitName,
-					noderesources.FitArgs{IgnoredResources: ignoredExtendedResources},
+					schedulerv1alpha2.NodeResourcesFitArgs{IgnoredResources: ignoredExtendedResources},
```

Comment:
why is this mixing internal config types (NewPluginConfig) with external config types (schedulerv1alpha2.NodeResourcesFitArgs)?

Reply:
This is temporary. The next step is to make the plugins only handle the internal types. For that, we need to do the conversion of user provided args into the internal types during the initial parsing of component config, as described in https://github.com/kubernetes/kubernetes/issues/87784#issuecomment-592709524

Reply:
The idea behind this PR here was to make the plugin arg types versioned. One can argue that the args of the internal scheduler plugins are a part of the API and therefore should be versioned. The conversion was supposed to be addressed in the v1.19 release cycle.

Reply:
talked through the plan for shifting plugins to internal types with @alculquicondor, this is ok as an interim step